README.md

```diff
@@ -1,123 +1,90 @@
-feedback
-========
+# Feedback
 
-This app handles user feedback related things.
+This is a public facing Ruby on Rails application that collects feedback from users in multiple ways (which are detailed below). This feedback information is then used to raise tickets in Zendesk.
```

Comment:
Not all feedback is sent to zendesk.  It might be better to say something along the lines of "This feedback is then sent to the `support` app or `support-api` app to be dealt with."